sdk/containerapps/Azure.ResourceManager.AppContainers/src/Generated/Models/JobExecutionNamesCollection.cs

```diff
@@ -1,31 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
-
-#nullable disable
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Azure.Core;
-
-namespace Azure.ResourceManager.AppContainers.Models
```

Comment:
breaking change, a model is removed.

Reply:
This was not used - `JobExecutionNamesCollection`  and was removed.